app/tests/unit/components/realmAdminSvc.spec.js

```diff
@@ -280,6 +280,39 @@ describe('RealmAdminService getClientSecret', () => {
   });
 });
 
+describe('RealmAdminService generateNewClientSecret', () => {
+
+  it('should throw an error connection is not set', async () => {
+    const svc = new RealmAdminService(realmConfig);
+    svc.axios = undefined;
+    await expect(svc.generateNewClientSecret('1')).rejects.toThrow();
+  });
+  it('should throw an error connection is not set', async () => {
```

Comment:
These it description strings are the same between the two tests.

Reply:
Fixed